### DISCUSSION TITLE
Please Add: Unicode-Name … Thanks!

### DISCUSSION THREAD

**Discussion by WildcatMidnight** [shriprem/Goto-Line-Col-NPP-Plugin#38] [General]
Hi! 
 
This is a useful plugin and especially for me because I am interested in the Unicode code point feature at the cursor position of the character. 
And it would be great and even more useful if you could please enhance that feature by integrating the Unicode-Name of the character/symbol in addition. That would be very meaningful and helpful for the community of NPP users who work with/are interested in Unicode in general. 
 
Thank you very much in advance!

**Comment by shriprem** [comment]
@WildcatMidnight , that is an excellent idea!

I have begun looking at possible ways to implement this. I will let you know when the plugin is ready with this new feature. Thank you again.

**Comment by shriprem** [comment]
The challenge of implementing this feature turned out to be interesting enough that I burned some midnight oil and got it done. I was able to include the 'Unicode Block' and the 'Unicode Name' data in the 'Cursor Position Data' field on the plugin panel. This data is also replicated in the calltip that pops up when using the plugin panel field controls to navigate a file.

During my testing, the new feature worked well for most character blocks except for some emojis and far eastern character sets like Hangul.

You can try out this new pre-release feature. 
1. Download one of the zip file versions below to match your Notepad++ bit version.
[GotoLineCol_x86.zip](https://github.com/user-attachments/files/17385195/GotoLineCol_x86.zip) (32-bit)
[GotoLineCol_x64.zip](https://github.com/user-attachments/files/17385203/GotoLineCol_x64.zip) (64-bit)
2. Extract the zip file contents into the `<NPP_Plugins_folder>/GotoLineCol` folder.
